runner/src/exercism_test_runner/internal.gleam

```diff
@@ -21,7 +21,7 @@ pub type Error {
 }
 
 pub type Suite {
-  Suite(name: String, path: String, tests: List(Test))
+  Suite(name: String, path: String, tests: List(the_test))
```

Comment:
This looks like it should raise a type error, here `Test` was a concrete type, replacing it with `the_test` (with a lowercase) makes it a type variable; however the type definition doesn't have any. Could you revert this change?

Reply:
Done in [b4c1f50](https://github.com/exercism/gleam-test-runner/pull/44/commits/b4c1f5072f7477cc77933819ce10f47f76eb1985)